src/docs/advanced/use-sealer-rootless.md

```diff
@@ -0,0 +1,27 @@
+# Run sealer in non-root mode
+
+## Install sealer using sudo
+
+**Resolved remote read permission to `sshd_config` file:**
```

Comment:
do you mean we need to check the permission of all nodes? if not 644, need chmod? if so,we can describe it more clearly .

Reply:
yes,Only the current running node needs to execute local sshd permissions.So I didn't tell them to configure on all nodes